applications/roloi-multisig-vault.md

```diff
@@ -0,0 +1,172 @@
+# Roloi Multi-Sig Vault
+
+- **Team Name:** NeoPower Digital
+- **Project Category:** Canary DApp
+- **Payment Address:** `1dz667uQX199rHyj6tizmL6EJe4AZxjH1RhnyrT1QuQ4pfS`
+
+## Project Overview :page_facing_up:
+### Overview
+
+Multi-Sig Vault Canary Dapp. 
+
+### Project Details
```

Comment:
Could you describe some cryptographic details of your solution? For example, how is the address of a vault generated?

Reply:
Then users will have to pay for txs they want to cosign instead of deposits in native multisig, right?

Reply:
Exactly. On this version, the signer that executes the transaction will have to pay it (after the transaction is properly approved).